cnd/src/http_api/routes/get.rs

```diff
@@ -0,0 +1,25 @@
+use crate::{
+    http_api::problem,
+    swap_protocols::{Facade, LocalSwapId},
+};
+use http_api_problem::HttpApiProblem;
+use warp::{Rejection, Reply};
+
+pub fn into_rejection(problem: HttpApiProblem) -> Rejection {
+    warp::reject::custom(problem)
+}
+
+#[allow(clippy::needless_pass_by_value)]
```

Comment:
Is my understanding correct that this will go away once we actually do the loading?

Reply:
No I don' think so. The problem is that `warp` passes us this by value so we actually cannot change the fn signature but clippy doesn't know that :(